### PR TITLE
Squash http block in zipkin receiver

### LINKED ISSUE #2997: Flow: squash `http` in `otelcol.receiver.zipkin`
The `http` block in `otelcol.receiver.zipkin` should be squashed to match the squashing used in the upstream config.

### PR DESCRIPTION
**Pull request by ptodev**
#### PR Description
Squashes the http block in zipkin receiver.

#### Which issue(s) this PR fixes
Closes #2997

#### PR Checklist

- [x] CHANGELOG updated
- [x] Documentation added
- [x] Tests updated

### CODE REVIEW

**Review by erikbaranowski — APPROVED** [review]
LGTM!

**Review by rfratto — APPROVED** [review]
LGTM, thanks! Once the merge conflict is resolved we can get this in.

**Pull request merged**
